ArduPlane/quadplane.cpp

```diff
@@ -2223,6 +2243,10 @@ void QuadPlane::run_xy_controller(float accel_limit)
         pos_control->init_xy_controller();
     }
     pos_control->set_lean_angle_max_cd(MIN(4500, MAX(accel_to_angle(accel_limit)*100, aparm.angle_max)));
+    if (q_fwd_throttle > 0.95f) {
```

Comment:
Should we check tilt too? It will probably hit tilt max before forward throttle is at max.

Reply:
That shouldn't be needed because the position controller is handling that limiting internally using the lean angle limit set by the call to set_lean_angle_max_cd in the line above.